js/clipboard.js

```diff
@@ -182,39 +182,39 @@
         return;
       }
 
+      let $form = $("<form />", {
+        "id": "sendForm",
+        "action": $sendButton.data("url"),
+        "method": $sendButton.data("method"),
+      });
+
       // Generate clipboard send data
       var data = {base_url: $sendButton.data('site-base-url')};
+      $form.append(`<input name='base_url' type='hidden' value='${data.base_url}'/>`);
```

Comment:
Nitpick: I like the way you created the form element, I think it's cleaner than a long string and I'd do the same for the inputs.